cmd/state/internal/cmdtree/invite.go

```diff
@@ -31,13 +34,18 @@ func newInviteCommand(prime *primer.Values) *captain.Command {
 		},
 		[]*captain.Argument{
 			{
-				Name:        "email1,[email2,..]",
+				Name:        "email1[,email2,..]",
 				Description: locale.Tl("invite_arg_emails", "Email addresses to send the invitations to"),
 				Required:    true,
 				Value:       &params.EmailList,
 			},
 		},
-		func(ccmd *captain.Command, _ []string) error {
+		func(ccmd *captain.Command, args []string) error {
```

Comment:
I am going back and forth on this but overall I think I'd personally prefer having this logic in the runner rather than the cmdtree. The idea is that cmdtree is all about command registration, and the runner is all about running the command.

Admittedly arg parsing does feel like a bit of a grey area. Perhaps worth for us to discuss during stand or weekly and have a stronger opinion on going forward.

Reply:
I think you're right. I'll move it.